### PR TITLE
Link wiki response in README.md

### PR DESCRIPTION
**Pull request by KaanOzkan**
Copied Ufuk's answer from this issue https://github.com/Shopify/tapioca/issues/79 into a wiki https://github.com/Shopify/tapioca/wiki/How-does-tapioca-compare-to-%22srb-rbi-gems%22-%3F

This PR creates a new header in the README.md to link to the wiki answer

### CODE REVIEW

**Comment by Morriar** [comment]
I think this should come sooner in the documentation and maybe not as a link but as text directly.

Most of the people that come to the `tapioca` repo have already eared about `srb` and might have the same question: "why should I use `tapioca` instead?". I think we need to answer this before even explaining how to install it.

Maybe another Idea would be to extract a short bullet list from the wiki post and put it int the introduction of the tool with a link to the post for more details?

**Comment by Morriar** [comment]
It might also be useful to add this explanation in our internal docs: https://development.shopify.io/engineering/keytech/reference/sorbet/other_projects/

**Comment by KaanOzkan** [comment]
It makes sense to me to put it earlier in the readme but I think it's too large of an answer to put the text in directly. 

Also I don't know how effectively we can extract it into a short bullet list. Because in my opinion if it's going to come before `installation` it needs to be quarter of its current length. I don't want to lose information/accuracy because chances are people won't go to the wiki if a summary is on the readme. But if everyone thinks it's possible to explain the points in a concise way then let's try it.

**Comment by Morriar** [comment]
> Also I don't know how effectively we can extract it into a short bullet list. Because in my opinion if it's going to come before `installation` it needs to be quarter of its current length.

Good point! What do you think about moving the current section as it is right now just before `## Installation`?

**Comment by KaanOzkan** [comment]
> What do you think about moving the current section as it is right now just before ## Installation?

I think thats better. I wasn't aware of how common/important this question was.

**Comment by paracycle** [comment]
I also agree that this is an important question in people's minds about what makes this different from `srb` tooling, so it would be great to have it more prominently in the `README`.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ If you ever run into a case, where you add a gem or update the version of a gem
 You can use the command `tapioca require` to auto-populate the `sorbet/tapioca/require.rb` file with all the requires found
 in your application. Once the file generated, you should review it, remove all unnecessary requires and commit it.
 
+## How does tapioca compare to "srb rbi gems" ?
```
Comment (Morriar) [inline README.md:37]:
```suggestion
## How does tapioca compare to "srb rbi gems" ?

```

To be consistent with the style in the rest of the document.

**Pull request merged**